dlc-node/src/main/scala/org/bitcoins/dlc/node/DLCNode.scala

```diff
@@ -79,4 +72,33 @@ case class DLCNode(wallet: DLCWalletApi)(implicit
       accept
     }
   }
+
+  override def sendDLCOffer(
+      peerAddress: InetSocketAddress,
+      message: String,
+      offerTLV: DLCOfferTLV): Future[Unit] = {
+    for {
+      handler <- connectToPeer(peerAddress)
+    } yield {
+      val peer = NormalizedString(
+        peerAddress.getHostString + ":" + peerAddress.getPort)
+      val msg = NormalizedString(message)
+      val lnMessage = LnMessage(
+        DLCOfferMessageTLV(peer = peer, message = msg, offer = offerTLV))
```

Comment:
We want our peer address injected here sending the message to our friend so they can get back to us.